package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "maplibre-gl-compare",
+  "name": "@maplibre/maplibre-gl-compare",
   "version": "0.5.0",
   "description": "Swipe and sync between two Maplibre maps",
```

Comment:
Please change `Maplibre` to `MapLibre`

Reply:
Which Maplibre exactly? 

Shouldn't that be consistent? Here is also in lower case: https://github.com/maplibre/maplibre-gl-leaflet/blob/main/package.json

Reply:
Sorry for not being clear. I am referencing the `description`, swipe an sync between.... The `name` is fine as it is now.